scrape.go

```diff
@@ -1,16 +1,18 @@
 package main
 
 import (
-	"io/ioutil"
+	"io"
 	"net/http"
 )
 
+const userAgent = "planetgolang.dev/1.0"
```

Comment:
Made it a const, partly since it won't change.

But mostly because when I test the `http.Client`, that I can check for the User-Agent.